lib/ansible/module_utils/acme.py

```diff
@@ -230,27 +230,27 @@ def _parse_key_openssl(openssl_binary, module, key_file=None, key_content=None):
         if asn1_oid_curve == 'prime256v1' or nist_curve == 'p-256':
             bits = 256
             alg = 'ES256'
-            hash = 'sha256'
+            hashalg = 'sha256'
             point_size = 32
             curve = 'P-256'
         elif asn1_oid_curve == 'secp384r1' or nist_curve == 'p-384':
             bits = 384
             alg = 'ES384'
-            hash = 'sha384'
+            hashalg = 'sha384'
             point_size = 48
             curve = 'P-384'
         elif asn1_oid_curve == 'secp521r1' or nist_curve == 'p-521':
             # Not yet supported on Let's Encrypt side, see
             # https://github.com/letsencrypt/boulder/issues/2217
             bits = 521
             alg = 'ES512'
-            hash = 'sha512'
+            hashalg = 'sha512'
             point_size = 66
             curve = 'P-521'
         else:
             return 'unknown elliptic curve: %s / %s' % (asn1_oid_curve, nist_curve), {}
-        bytes = (bits + 7) // 8
-        if len(pub_hex) != 2 * bytes:
+        no_bytes = (bits + 7) // 8
```

Comment:
How about `num_bytes`? `no` sounds a bit "negative" to me.

Reply:
Sure, fine for me.